truffle/test/LoadContract_Escrow.js

```diff
@@ -334,20 +336,20 @@ contract('LoadContract with Escrow', async (accounts) => {
         assert.equal(data.shipment.state, ShipmentState.CREATED);
         assert.equal(data.escrow.state, EscrowState.FUNDED);
 
-        await contract.setEscrowRefundAddress(shipmentUuid, INVALID, {from: OWNER});
+        await contract.setEscrowRefundAddress(shipmentUuid, ATTACKER, {from: OWNER});
 
         await truffleAssert.reverts(contract.refundEscrow(shipmentUuid, {from: SHIPPER}), "Refunds can only be issued to Canceled shipments by the Moderator");
         await contract.refundEscrow(shipmentUuid, {from: OWNER});
 
         await truffleAssert.reverts(contract.withdrawEscrow(shipmentUuid, {from: SHIPPER}), "Escrow can only be withdrawn by carrier if released or by shipper if refunded");
 
-        let invalidBalance = web3.utils.toBN(await web3.eth.getBalance(INVALID));
-        let withdrawTxReceipt = await contract.withdrawEscrow(shipmentUuid, {from: INVALID});
+        let invalidBalance = web3.utils.toBN(await web3.eth.getBalance(ATTACKER));
+        let withdrawTxReceipt = await contract.withdrawEscrow(shipmentUuid, {from: ATTACKER});
         const withdrawTx = await web3.eth.getTransaction(withdrawTxReceipt.tx);
         const gasCost = web3.utils.toBN(withdrawTx.gasPrice).mul(web3.utils.toBN(withdrawTxReceipt.receipt.gasUsed));
         data = await getShipmentEscrowData(shipmentUuid);
         assert.equal(data.escrow.state, EscrowState.WITHDRAWN);
-        expect(web3.utils.toBN(await web3.eth.getBalance(INVALID))).to.eq.BN(invalidBalance.add(web3.utils.toBN(web3.utils.toWei("1.0", "ether"))).sub(gasCost));
+        expect(web3.utils.toBN(await web3.eth.getBalance(ATTACKER))).to.eq.BN(invalidBalance.add(web3.utils.toBN(web3.utils.toWei("1.0", "ether"))).sub(gasCost));
```

Comment:
I know web3 is super ugly to work with, but let's not make it worse with this Franken-line; in general I'd prefer we split things like this into 3-4 lines for clarity.

Reply:
Do you mean using some more local variables and therefore to 3-4 line of statements like the following or only splitting one statement to multiple lines? 
`
        const updatedBalance = await web3.utils.toBN(await web3.eth.getBalance(ATTACKER));
        const balanceAfterAdd = invalidBalance.add(web3.utils.toBN(web3.utils.toWei("1.0", "ether")));
        const balanceAfterSub = balanceAfterAdd.sub(gasCost);
        expect(updatedBalance.to.eq.BN(balanceAfterSub));
`

Reply:
An example of spiting the lines without more statements. 
![image](https://user-images.githubusercontent.com/1012073/63179941-348fcc80-c01b-11e9-81c0-4f4f3a014dc9.png)

Reply:
or like this?
![image](https://user-images.githubusercontent.com/1012073/63180837-13c87680-c01d-11e9-8729-8a03b3b173b3.png)

Reply:
I think the first example is the cleanest to look at!